packages/expo-module-template-local/ios/{%- project.name %}.podspec

```diff
@@ -0,0 +1,24 @@
+require 'json'
+
+Pod::Spec.new do |s|
+  s.name           = '<%- project.name %>'
+  s.version        = '0.0.1'
+  s.summary        = 'A sample project summary'
+  s.description    = 'A sample project description'
+  s.author         = ''
+  s.homepage       = 'https://expo.dev/'
+  s.platform       = :ios, '13.0'
```

Comment:
Following the podspec syntax reference, it looks like we can remove the platform:
> If not defined in a subspec the attributes of this group inherit the value of the parent.

Generally I would remove every attribute that is not required (better if the user doesn't have to modify the podspec, e.g. when bumping the iOS deployment target in the app)